app/utils/lib/isURL.js

```diff
@@ -1 +1 @@
-export const isURL = (str) => /^https?:\/\//.test(str);
+export const isURL = (str) => /^(https?:\/\/|data:)/.test(str);
```

Comment:
It works now, but the name of the constant is a little off now, since it doesn't get only URLs now but also data *URIs*

```suggestion
export const isURLorData = (str) => /^(https?:\/\/|data:)/.test(str);
```

Maybe this? 🤔

Reply:
You know what is wierd? [Read the spec.](https://tools.ietf.org/html/rfc2397)

Reply:
Also, [this](https://url.spec.whatwg.org/#goals):

> Standardize on the term URL. URI and IRI are just confusing. In practice a single algorithm is used for both so keeping them distinct is not helping anyone. URL also easily wins the search result popularity contest.

Reply:
Holy shoe! I thought it was called data URI scheme :eyes: